protocol-configuration/mainnetAddresses.ts

```diff
@@ -1983,6 +1993,11 @@ const MainnetContractsConfig: MainnetContractsConfig = {
     address: '0xf2e513d3b4171bb115cb9ffc45555217fbbbd00c',
     category: AddressCategory.Turbo
   },
+  convexPoolPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0x525eA5983A2e02abA8aA0BE7D15Cd73150812379',
+    category: AddressCategory.PCV_V1
```

Comment:
fun fact but this PCV deposit is actually using the PCV_V2 interface, idk why but a wrapper has been deployed, and it's the wrapper that is in the CR oracle (even though there was no need for wrapping)

If we flag this as PCV, the e2e automatically expect it to be configured in the CR oracle, which is not the case

Reply:
I added it because it was not in our mainnetAddresses file & that's a problem because it's holding protocol funds (only the wrapper was in mainnetAddresses)

Reply:
That's so weird